image-build-push.sh

```diff
@@ -0,0 +1,36 @@
+# Terminal output colors 
```

Comment:
Should be moved into the `scripts` folder. Seems to just be a nicer build-image.sh so you can probably just swap it 1:1. Or maybe the purpose is to have the confirm version be in a seperate file so the CI can work well?

Reply:
Moved into `scripts` folder.